apps/mobile/src/features/balances/stacks/mock-signers.ts

```diff
@@ -0,0 +1,62 @@
+const multiWalletSigners = [
```

Comment:
This is a log of the signers list for:
- Leather dev wallet: `fingerprint: 'efd01538'`
- my wallet : `fingerprint: '558efa7e'`

I imported the dev wallet and had one new account created with `accountIndex: 0`. 

On adding a new account to that wallet in the UI I can see it added with a new derivation path but the `accountIndex` is always `0`.